### PR TITLE
FF-243 for  FF-218

### PR DESCRIPTION
**Pull request by qvalentin**

### CODE REVIEW

File: webapp_frontend/src/background/constants.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const prod: constantsdef = {
 const dev: constantsdef = {
     url: {
         //API_URL: 'https://cors.filefighter.de/http://filefighter.de/api',
-       // API_URL: 'https://cors-anywhere.herokuapp.com/http://filefighter.ddns.net:3001/api',
+        // API_URL: 'https://cors-anywhere.herokuapp.com/http://filefighter.ddns.net:3001/api',
         API_URL: 'http://localhost:8080',
```
Comment (open-schnick) [inline webapp_frontend/src/background/constants.ts]: Intentionally?

Comment (qvalentin) [reply webapp_frontend/src/background/constants.ts]: yes, the new version is not on the server, or is it?

Comment (open-schnick) [reply webapp_frontend/src/background/constants.ts]: not yet. I will merge the pr soon. so the endpoint can be used.

Comment (open-schnick) [reply webapp_frontend/src/background/constants.ts]: Pr is merged, be sure to test it.

**Review by Gimleux — APPROVED** [review]

**Pull request merged**